recipe/bld.bat

```diff
@@ -0,0 +1,21 @@
+@echo on
+
+set "ARGS="
+
+if not "%CUDA_COMPILER_VERSION%" == "None" (
+    set "NVCC_APPEND_FLAGS=%NVCC_APPEND_FLAGS% --use-local-env"
+    set ARGS=--cuda_path="%CUDA_HOME%"
+    if "%CUDA_COMPILER_VERSION%" == "12.0" (
+        set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUDA_COMPILER_VERSION%
+        set CUDA_BIN_PATH=%CUDA_PATH%\bin
+        @REM set CUDA_TOOLKIT_INCLUDE_DIR=%PREFIX%\Library
```

Comment:
```suggestion
```

Reply:
Thanks, done in https://github.com/conda-forge/warp-lang-feedstock/pull/13/commits/65c5514bda55a81e6f508dbf769673f381f88c88